src/components/UI/Button/Button.module.css

```diff
@@ -63,6 +63,11 @@
     font-size: 1rem;
 }
 
+.Page404 {
+    padding: 7px 50px;
```

Comment:
Rem is usually a better measurement than px due to accessibility. Change 50px to rem